.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,32 @@
+name: Pull Request Checks
```

Comment:
The existing `ip-verify.yml` use an action instead.
Here: `Check the pull request`